commands/chassis/ChassisMove.java

```diff
@@ -76,14 +76,16 @@ protected void initialize() {
 	protected void execute() {
 		chassis.moveCartesian(controller.getX(), controller.getY(), controller.getTurnSpeed());
 		motorSpeeds = chassis.getMotorSpeeds();
-		String motorSpeedsString = "";
+		StringBuilder motorSpeedsString = new StringBuilder();
+		motorSpeedsString.append("Motor speeds: ");
 		for (int i = 0; i < motorSpins.length; i++) {
 			LogKitten.d(Double.toString(motorSpeeds[i]));
 			motorSpins[i].set(motorSpeeds[i]);
-			motorSpeedsString += Double.toString(motorSpeeds[i]) + " ";
+			motorSpeedsString.append(motorSpeeds[i]);
+			motorSpeedsString.append(' ');
```

Comment:
To prevent a trailing space on the string, I would move this before the previous line and remove the trailing space  on line 80.